docs/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,15 @@
+export const Footer = () => {
+  return (
+    <footer
+      className={`not-prose sticky top-0 z-50 flex flex-wrap items-center justify-between bg-white px-4 py-5 shadow-md shadow-slate-900/5 transition duration-500 dark:bg-slate-900/95 dark:shadow-none dark:backdrop-blur sm:px-6 lg:px-8 dark:[@supports(backdrop-filter:blur(0))]:bg-slate-900/75`}
```

Comment:
is prettier ignoring these long lines?

Reply:
Seems to be. The tailwind prettier plugin is at least sorting the classNames according to logical groupings (i.e. flexbox stuff, color stuff, padding, etc). 

[This issue](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/33) claims that not breaking lines is  how its supposed to be, a claim which is incompatible with [this issue](https://github.com/prettier/prettier/pull/7865#issuecomment-836345253) which discusses how splitting classNames to 1 per line is not exactly an obvious win for readability. Manually inserted line breaks are preserved, however, and className auto-sorting still works on the broken up lines. Manual fixes don't really scale, but it's something.

Always down for forking the tailwind prettier plugin so we can do it correctly. I ♥ scope creep.